bentoml/server/bento_api_server.py

```diff
@@ -83,12 +84,16 @@ def __init__(self, bento_service, port=_DEFAULT_PORT, app_name=None):
 
         self.port = port
         self.bento_service = bento_service
+        self.app = Flask(app_name, static_folder=None)
 
-        self.app = Flask(
+        self.static_path = os.path.join(
+            self.bento_service._bento_service_bundle_path,
```

Comment:
`_bento_service_bundle_path` is the only set when loading from a saved BentoService bundle.

Maybe you could add a method `get_web_static_content_path`:

```
def get_web_static_content_path(self):
   if self.bento_service._bento_service_bundle_path:
      return os.path.join(
            self.bento_service._bento_service_bundle_path, 'web_static_content'
      )
    elif: bento_service.web_static_content:
        return os.path.join(os.getcwd(), bento_service.web_static_content)
    else:
        return None / raise ?

```

Reply:
This should fix the error you see in the unit test, it will also allow the user to test the BentoAPIServer without saving and loading, this can be super useful for debugging in an interactive session or Jupyter notebook

Reply:
This is perfect!!
I wanted something like this but was not sure how to implement it😅